src/typed-method-types/workflows/triggers/event.ts

```diff
@@ -57,6 +60,8 @@ type ChannelEvents =
   & {
     /** @description The channel id's that this event listens on */
     channel_ids: PopulatedArray<string>;
+    // deno-lint-ignore no-explicit-any
+    [otherOptions: string]: any;
```

Comment:
These are intentionally strewn across trigger inputs in the PR. The intent is to make sure that if triggers start to accept new data that we don't expect, they are still usable without typeahead.